src/main/target/KISSFC/target.h

```diff
@@ -21,7 +21,7 @@
 
 #define CONFIG_FASTLOOP_PREFERRED_ACC ACC_NONE
 
-#define SBUS_PORT_OPTIONS (SERIAL_STOPBITS_2 | SERIAL_PARITY_EVEN | SERIAL_INVERTED | SERIAL_BIDIR)
+#define SBUS_PORT_OPTIONS (SERIAL_STOPBITS_2 | SERIAL_INVERTED | SERIAL_BIDIR)
```

Comment:
This seems to be entirely redundant and should be removed to make target maintenance easier.

Reply:
No idea why it's there, I'll research when doing changes for 3.3

Reply:
A lot of targets seem to be re-defining defaults in `config.c`, making maintenance harder and leading to faulty targets when defaults are changed. :disappointed: